docs/Developers/Development_Setup.rst

```diff
@@ -40,18 +40,29 @@ Using venv
 Using virtualenv
 ~~~~~~~~~~~~~~~~
 
-- Install virtualenv using pip :
+- Install virtualenv using pip3 :
     ::
 
-        $ pip install virtualenv
+        $ pip3 install virtualenv
 
 - Create the virtualenv :
     ::
 
         $ cd working_dir # move into the dir where you want to create coala-venv
         $ virtualenv coala-venv
 
-- Run coala-venv :
+-NOTE:
+ If you have both Python3 and Python2 installed check this out first.
+
+::
+
+    $ virtualenv coala-venv -p $(which python3)
+
+-This creates a isolated Python3 environment called coala-venv
+ In your current directory.
+ As coala only works for Python >= 3.4.
```

Comment:
`In` -> `in`
These two sentences can be combined into one, in my opinion.
```
This creates a isolated Python3 environment called coala-venv In your current directory. As coala only works for Python >= 3.4
```
can become
```
This creates an isolated Python3 environment called coala-venv in your current directory, as coala only works for Python >= 3.4.
```